stl/inc/__msvc_chrono.hpp

```diff
@@ -666,18 +666,53 @@ namespace chrono {
         using time_point                = _CHRONO time_point<steady_clock>;
         static constexpr bool is_steady = true;
 
+#if defined(_M_ARM) || defined(_M_ARM64) // vvv ARM/ARM64 arch vvv
+
+#if _HAS_CXX20
+#define _LIKELY_ARM [[likely]]

```

Comment:
I don't think we should make separation of general `[[likely]]` macroization, where we map to `[[likely]]`/`[[__likely__]]`/nothing and the x86 / ARM selection.

The former macroization might go to `yvals_core.h`, where other attributes are macroized too.

This way we can re-use `_LIKELY` / `_UNLIKELY` in other places too